gatsby-config.js

```diff
@@ -25,6 +25,22 @@ module.exports = {
         configPath: require.resolve(`./i18n/config.json`),
       },
     },
+    {
+      resolve: `gatsby-source-airtable`,
+      options: {
+        apiKey: `keydsUE1ihWxveV3G`,
```

Comment:
API keys sollten eigentlich auf keinen Fall im Repo selber auftauchen. Das sollte über Environment Variablen laufen (s. z.B. https://www.gatsbyjs.com/docs/how-to/local-development/environment-variables/ wie gatsby das vorschlägt zu machen).